user_docs/en/userGuide.t2t

```diff
@@ -2580,63 +2581,100 @@ To toggle the braille viewer from anywhere, please assign a custom gesture using
 The NVDA Python console, found under Tools in the NVDA menu, is a development tool which is useful for debugging, general inspection of NVDA internals or inspection of the accessibility hierarchy of an application.
 For more information, please see the [NVDA Developer Guide https://www.nvaccess.org/files/nvda/documentation/developerGuide.html].
 
-++ Add-ons Manager ++[AddonsManager]
-The Add-ons Manager, accessed by selecting Manage add-ons under Tools in the NVDA menu, allows you to install, uninstall, enable and disable add-on packages for NVDA.
+++ Add-on Store ++[AddonsManager]
+The Add-on Store allows you to browse and manage add-on packages for NVDA.
 These packages are provided by the community and contain custom code that may add or change features in NVDA or even provide support for extra Braille displays or speech synthesizers.
+The Add-on Store is accessed by the Tools submenu in the NVDA menu.
+To access the Add-on Store from anywhere, please assign a custom gesture using the [Input Gestures dialog #InputGestures].
 
-The Add-ons Manager contains a list that displays all the add-ons currently installed in your NVDA user configuration. 
-Package name, status, version and author are shown for each add-on, though further information such as a description and URL can be viewed by selecting the add-on and pressing the About add-on button.
-If there is help available for the selected add-on, you can access it by pressing the Add-on help button.
++++ Browsing add-ons +++[AddonStoreBrowsing]
+The Add-on Store displays a list of add-ons.
+You can jump to the list with ``alt+l``.
+If you have not installed an add-on before, the add-on store will open on a list of add-ons available to install.
+If you have installed add-ons, the list will display currently installed add-ons.
 
-To browse and download available add-ons online, press the Get add-ons button.
-This button opens the [NVDA Add-ons page https://addons.nvda-project.org/].
-If NVDA is installed and running on your system, you can open the add-on directly from the browser to begin the installation process as described below.
-Otherwise, save the add-on package and follow the instructions below.
+You can sort add-ons by their various fields by pressing ``ctrl+1,2,3...``, where the number is the column index you wish to sort by.

```

Comment:
These actions don't give user feedback when sorting in the GUI. While this is UX related, I'd suggest implementing sort feedback first before publicly announcing this feature in the documentation.

Reply:
How do you suggest giving feedback for these sorts?

Reply:
I would also encourage this sort of feedback to be provided via https://github.com/nvaccess/nvda/discussions/14912

Reply:
>One concern I have, is that starting a build of NVDA with the add-on store in place, then starting a build without the add-on store seems to generate a new addons state, as disabled add-ons are enabled again. While downgrading is discouraged, I think this should at least be documented as known behavior.

Also responding to this here, please raise this in https://github.com/nvaccess/nvda/discussions/14912.
Handling downgrading is definitely out of scope of this PR

Reply:
I was originally very confused with this feature. As I would have expected that ctrl+1 sorted by the first column (I.e. the add-on's name) however I think ctrl+1 matches version? And ctrl+0 sorts by name?  I also agree that this feature could be better if  the sorting was exposed somehow. My feeling is that these shortcuts should be removed from the user guide before this is merged. And consider perhaps before 2023.2 adding another combo box for sorting.